bin/heroku-setup

```diff
@@ -2,7 +2,4 @@
 
 bundle exec rails db:schema:load:with_data
 bundle exec rails db:seed
-
-if [ ! -z "${SUPERADMIN_PASSWORD}" ]; then
-  bundle exec rake create_admin_user[admin,admin@simple.org,${SUPERADMIN_PASSWORD}]
-fi
+bundle exec rails db:seed_user_data
```

Comment:
🐞 typo: `seed_users_data`

Not sure if this is being done somewhere else in the setup but `db:seed_users_data` expects sidekiq and the app to be running.

Reply:
I'm guessing that Heroku will provision these resources before running this script. We'll find out 😄

Reply:
It works!